inbc-program/inbc/broker.py

```diff
@@ -23,8 +22,6 @@
 from inbm_vision_lib.mqttclient.mqtt import MQTT
```

Comment:
There is a DEVICE_STATUS_CHANNEL imported from inbm_vision_lib.constants. I think this is no longer needed.